airbyte-config/persistence/src/main/java/io/airbyte/config/persistence/DbConverter.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright (c) 2021 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.config.persistence;
+
+import static io.airbyte.db.instance.configs.jooq.Tables.CONNECTION;
+
+import io.airbyte.commons.enums.Enums;
+import io.airbyte.commons.json.Jsons;
+import io.airbyte.config.JobSyncConfig.NamespaceDefinitionType;
+import io.airbyte.config.ResourceRequirements;
+import io.airbyte.config.Schedule;
+import io.airbyte.config.StandardSync;
+import io.airbyte.config.StandardSync.Status;
+import io.airbyte.protocol.models.ConfiguredAirbyteCatalog;
+import java.io.IOException;
+import java.util.List;
+import java.util.UUID;
+import org.jooq.Record;
+
+public class DbConverter {
+
+  public static StandardSync buildStandardSync(final Record record, final List<UUID> connectionOperationId) throws IOException {
```

Comment:
I like this refactor to have the converters. Is this code the same as the code in `DatabaseConfigPersistence` [here](https://github.com/airbytehq/airbyte/blob/master/airbyte-config/persistence/src/main/java/io/airbyte/config/persistence/DatabaseConfigPersistence.java#L659-L677)? Is it worth having the `DatabaseConfigPersistence` depend on this too?

Reply:
This is correct. Yes I can have DatabaseConfigPersistence depend on that class instead

Reply:
This is done